microcosm_flask/errors.py

```diff
@@ -101,7 +101,13 @@ def extract_headers(error):
     Extract HTTP headers to include in response.
 
     """
-    return getattr(error, "headers",  {})
+    try:
```

Comment:
https://stackoverflow.com/questions/11360858/what-is-the-eafp-principle-in-python

Reply:
Also note that your proposed fix would invert the order: I'm checking for `headers` first before `get_headers` since this minimizes backwards incompatibility.